wellcomeml/ml/spacy_classifier.py

```diff
@@ -109,8 +112,19 @@ def fit(self, X, Y):
             #dropout = decaying(0.6, 0.2, 1e-4)
             for i in range(n_iter):
                 losses = {}
-                # batch up the examples using spaCy's minibatch
-                random.shuffle(train_data)
+
+                def shuffle(X_train, Y_train):
```

Comment:
any reason why shuffle is defined here and not as a @classmethod ?

Reply:
not really, I have seen this patter in spacy here and there and using it. not sure when it is good to define functions on the fly vs not. I guess if shuffle gets reused, it would definitely move to a class method